client/src/setupTests.js

```diff
@@ -0,0 +1,35 @@
+//functions
+export const mockMapOn = jest.fn();
+export const mockRemove = jest.fn();
+export const mockDisable = jest.fn();
+export const mockMapdragRotate = jest.fn();
+export const mockMapaddControl = jest.fn();
+export const mocktouchZoomRotate = jest.fn();
+export const mockdoubleClickZoom = jest.fn();
+export const mockgetWest = jest.fn()
+
+//constructors
+export class mockLngLat {
+};
+export class mockLngLatBounds {
+};
+export class mockNavigationControl {
+};
+
+jest.mock('mapbox-gl', () => ({
+    Map: function () {
+        this.on = mockMapOn;
+        this.remove = mockRemove;
+        this.dragRotate = mockMapdragRotate;
+        this.addControl = mockMapaddControl;
+        this.touchZoomRotate = mocktouchZoomRotate;
+        this.doubleClickZoom = mockdoubleClickZoom;
+
+        this.dragRotate.disable = jest.fn();
+        this.touchZoomRotate.disableRotation = jest.fn();
+        this.doubleClickZoom.disable = jest.fn();
+    },
+    LngLat: mockLngLat,
+    LngLatBounds: mockLngLatBounds,
```

Comment:
I'm noticing that although the tests are passing, there's a [console error in the logs](https://github.com/rachelekm/vegishphilly/pull/15/checks?check_run_id=3898417186#step:3:186) . I'm not clear why it's not causing the test to fail, but it should be fixed one way or another (looks like it just needs another mock).